compiler/include/resolution.h

```diff
@@ -29,7 +29,9 @@ class CallInfo;
 
 extern SymbolMap paramMap;
 extern Vec<CallExpr*> callStack;
-extern Map<Type*,FnSymbol*> autoCopyMap; // type to chpl__autoCopy function
+bool hasAutoCopyForType(Type* type);
```

Comment:
If we replace hasAutoCopyForType() with getAutoCopyOrNullForType(), which returns straight autoCopyMap[type], we can replace two calls hasAutoCopyForType+getAutoCopyForType with just a call to getAutoCopyOrNullForType.